### ISSUE TITLE
Suggestion: add the "Concepts" wiki page to the official documentation

### ISSUE THREAD

**Issue by Vrihub** [beetbox/beets#2277]
I really think the content of https://github.com/beetbox/beets/wiki/Concepts should be added to the official documentation, I wish I found it earlier! :)

These concepts can be quite non-obvious for the novice user, accustomed with the workflow of other music tagging tools (no separate database, edit/write metadata directly into the files, exit and let the user move those files around in the filesystem manually).

Possible locations?
  * as a new initial chapter in the "Guides" section?
  * at the beginning of the Guides -> Getting Started chapter?

**Comment by sampsyo** [comment]
Good idea! I wrote that some time ago now but never really expanded it beyond its initial version. Is there anything else we should include there?

**Comment by abigaildelarosa** [comment]
Hello, 

I take a look at this issue as well and as a new beets user considered highly important to add those concepts into the Getting started chapter. And as a suggestion for what else we can add to the file, I think it will be a great idea to include something related to the standard programming(from the developer point of view) that we should be follow. Do you have any suggestions about where should I take a look to add the standard information? Please let me know and will come up with a new file version.

**Comment by sampsyo** [comment]
Cool; thanks for weighing in. Out of curiosity, which concepts from that page did you find particularly helpful? I don't think we can add the *whole* content of the Concepts page to Getting Started, but it would make sense to include the basics.

I didn't quite understand what you meant by "standard programming"—any chance you could clarify?
